pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -57,11 +61,20 @@ def pytorch_funcify_FunctionGraph(
     conversion_func=pytorch_funcify,
     **kwargs,
 ):
+    def constants_wrapper(x, **kwargs):
+        x = pytorch_typify(x)
+
+        @torch.compiler.assume_constant_result
+        def torch_assume_constant(arg=x):
```

Comment:
this unfortunately suffers from the same problem some of the generated functions were getting where pytorch loses their reference somewhere in the process. We'll need to make sure it gets added to the `warpper::gen_functors` list